.github/workflows/nodejs.yml

```diff
@@ -13,32 +13,35 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x, 13.x]
         webpack-version: [next, latest]
 
     steps:
     - uses: actions/checkout@v1
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Using Node v${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: yarn install, bootstrap
+    - name: Install dependencies
       run: |
         yarn
         yarn bootstrap
-    - name: Run test for webpack version ${{ matrix.webpack-version }}
+    - name: Linting
+      run: yarn lint
```

Comment:
Let's move it to own step, we don't need run link on every platform and node version